karapace/protobuf/kotlin_wrapper.py

```diff
@@ -0,0 +1,59 @@
+from karapace.protobuf.exception import IllegalArgumentException, IllegalStateException
+
+import textwrap
+
+
+def check(q: bool, message: str) -> None:
+    if not q:
+        raise IllegalStateException(message)
+
+
+def trim_margin(s: str) -> str:
```

Comment:
please replace this with https://docs.python.org/3/library/textwrap.html?highlight=dedent#textwrap.dedent

Reply:
This function contains a bit more logic than textwrap.dedent() which actually is used inide. To replace it with dedent() I need to change the format of test data in unit tests. But it is a lot of work plus the format of data in these tests come from square/wire project and has the sense.